samples/RegularExpressionSample/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Text.RegularExpressions;
+using Microsoft.Data.Sqlite;
+
+namespace RegularExpressionSample
+{
+    class Program
+    {
+        static void Main()
+        {
+            var connection = new SqliteConnection("Data Source=:memory:");
+            connection.Open();
+
+            var createCommand = connection.CreateCommand();
+            createCommand.CommandText =
+            @"
+                CREATE TABLE user (
+                    name TEXT,
+                    bio TEXT
+                );
+
+                INSERT INTO user
+                VALUES ('Arthur', 'I put two spaces after my full stops.  Always.'),
```

Comment:
@ajcvickers - :trollface:

Reply:
This is a work of fiction. Names, characters, business, events and incidents are the products of the author’s imagination. Any resemblance to actual persons, living or dead, or actual events is purely coincidental.